src/app/screens/Enable/index.tsx

```diff
@@ -71,6 +85,14 @@ function Enable(props: Props) {
         </p>
 
         <ConfirmOrCancel label="Enable" onConfirm={enable} onCancel={reject} />
+
+        <a
+          className="underline mt-8 text-sm text-gray-500 dark:text-gray-400"
+          href="#"
+          onClick={block}
+        >
+          Don{`'`}t ask for this site again
```

Comment:
This seems a bit odd. Maybe just write "Do not"? Or use one of eslints suggestions, i.e. `&apos;`